src/ethereum/prague/vm/instructions/control_flow.py

```diff
@@ -169,3 +181,211 @@ def jumpdest(evm: Evm) -> None:
 
     # PROGRAM COUNTER
     evm.pc += 1
+
+
+def rjump(evm: Evm) -> None:
+    """
+    Jump to a relative offset.
+
+    Parameters
+    ----------
+    evm :
+        The current EVM frame.
+
+    """
+    # STACK
+    pass
+
+    # GAS
+    charge_gas(evm, GAS_RJUMP)
+
+    # OPERATION
+    pass
+
+    # PROGRAM COUNTER
+    relative_offset = int.from_bytes(
+        evm.code[evm.pc + 1 : evm.pc + 3], "big", signed=True
```

Comment:
Is it ever possible to get an `IndexError` here, or is this safe because of the EOF validation?